python/tskit/trees.py

```diff
@@ -2242,6 +2242,22 @@ def num_migrations(self):
         """
         return self._ll_tree_sequence.get_num_migrations()
 
+    @property
+    def oldest_time(self):
+        """
+        Returns the oldest time for any of the edges in this tree sequence.
+
+        :return: The oldest time in this tree sequence.
+        :rtype: float
+        """
+        # Edges are guaranteed to be listed in parent-time order, so we can just get the
+        # last one, assuming there are any edges (otherwise return None). This is O(1)
+        # compared to looking for the maximum node time.
+        if self.num_edges:
+            l, r, parent, child = self._ll_tree_sequence.get_edge(self.num_edges - 1)
```

Comment:
No point in using the low-level API here: 
```
edge = self.edge(self.num_edges - 1)
return self.node(edge.parent).time

Reply:
I get `AttributeError: 'TreeSequence' object has no attribute 'edge'`

Reply:
Ah, I see that method is missing. Would you mind adding it? (Follow the definition of 'node' and change the definition of the ``edges`` iterator to use the new method)

Reply:
Right. GTG now. Will do later